frontend/src/app/app-routing.module.ts

```diff
@@ -31,9 +31,11 @@ import { McBreadcrumbsModule } from 'ngx6-angular-breadcrumbs';
 import { UserResolver } from './user-resolver.service';
 import { AdminSeasonDatesComponent } from './trees/admin/season-dates/season-dates.component';
 import { AdminDistrictDatesComponent } from './trees/admin/district-dates/district-dates.component';
+import { AdminFeedbackReviewComponent } from './trees/admin/feedback-review/feedback-review.component';
```

Comment:
I'm not sure I like how the file path contains `.. /feedback-review/feedback-review ... ` here and below it's `.. /feedback/submit-feedback ...`. Gonna make them follow the same pattern.

Reply:
on further thought i think this is alright for now.